src/builder.rs

```diff
@@ -54,45 +114,57 @@ impl Builder {
         self
     }
 
+    /// The minimum latency that a message will take to transfer over a
+    /// link on the network.
     pub fn min_message_latency(&mut self, value: Duration) -> &mut Self {
         self.link.latency_mut().min_message_latency = value;
         self
     }
 
+    /// The maximum latency that a message will take to transfer over a
+    /// link on the network.
     pub fn max_message_latency(&mut self, value: Duration) -> &mut Self {
         self.link.latency_mut().max_message_latency = value;
         self
     }
 
+    /// The failure rate of connections on the network. This is how
```

Comment:
These are per message so the link is not actually broken. Think of packet loss. It happens to break a tcp connection since we don't have re-send capabilities yet. With UDP, this is pretty useful for testing network flakiness.